src/NServiceBus.Persistence.CosmosDB/SynchronizedStorage/Operation.cs

```diff
@@ -37,6 +37,84 @@ public virtual void Conflict(TransactionalBatchOperationResult result)
                 case HttpStatusCode.Conflict:
                 case HttpStatusCode.PreconditionFailed:
                     throw new TransactionalBatchOperationException("Concurrency conflict.", result);
+                case HttpStatusCode.Accepted:
```

Comment:
This change is not correct. All of these should fall through to the default case and throw.

Reply:
Done